### DISCUSSION TITLE
how to stop the objects inside group are scaled when the group is scaling

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#8696] [General]
how to stop the objects inside group are scaled when the group is scaling?
https://codesandbox.io/s/border-problem-of-group-wmh0xp?file=/src/components/HelloWorld.vue
![image](https://user-images.githubusercontent.com/41353266/218969257-f58234e0-dd30-4237-8666-37c35e213708.png)

**Comment by wuxingping** [comment]
i came up with  by resizing instead of scaling.but i donot know exactly what i should do .i am going to try to implement it first.

**Reply by ShaMan123** [reply]
POC mr works a bit. ml will need to use `_adjustObjectPosition`
https://codesandbox.io/s/border-problem-of-group-forked-mjwfev?file=/src/container.ts

**Reply by ShaMan123** [reply]
This is another thing I want to work on, layout origin of object

**Reply by wuxingping** [reply]
> This is another thing I want to work on, layout origin of object

i am very looking forward to it.please tell me at the first time if you finish.
